.gitignore

```diff
@@ -58,3 +58,6 @@ target/
 
 # Jupyter notebooks
 notebooks/
+
+# venv
+venv
```

Comment:
Nit: the last character in this file (and in others) is not an end-of-line character (github shows it as a red symbol)